frontend/src/components/Button/Button.styled.ts

```diff
@@ -0,0 +1,75 @@
+import styled, { css } from 'styled-components';
+import { Button as AntButton } from 'antd';
+import { NativeButtonProps } from 'antd/lib/button/button';
+
+interface StyledButtonProps extends NativeButtonProps {
+  variant?: 'primary' | 'secondary';
+  disabled?: boolean;
+  size?: 'small' | 'large';
+  ghost?: boolean;
```

Comment:
Ta nazwa mi nic nie mówi :)
Fajnie jakbym mógł bez wchodzenia w komponent wiedzieć już do czego służy

Reply:
Która nazwę masz na myśli? Variant?

Reply:
Dobra, widzę , że github pokazuje teraz więcej kodu przy zaznaczeniu jednej lnijki :)
`ghost?: boolean;`

Reply:
Ogolnie nazwe zapozyczylam z ant design, teraz juz obecnie spowrotem jest to po prostu props dla ant design.